apps/prairielearn/src/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql

```diff
@@ -232,6 +246,12 @@ WITH
         ELSE NULL
       END AS correct,
       s.feedback,
+      row_number() OVER (
+        PARTITION BY
+          v.id
+        ORDER BY
+          s.date
+      )::integer AS submission_number,
```

Comment:
This is used to construct filenames in the files archives, so I needed to add it. This was copied directly from `assessment_instance_files`.